internal/search/keyword/term_utils.go

```diff
@@ -0,0 +1,60 @@
+package keyword
+
+import (
+	"strings"
+	"unicode"
+
+	"github.com/kljensen/snowball"
+)
+
+func removePunctuation(input string) string {
+	lowerCase := strings.ToLower(input)
+	runes := []rune(lowerCase)
+
+	// Remove leading and trailing punctuation
+	start := 0
+	for ; start < len(runes) && unicode.IsPunct(runes[start]); start++ {
+	}
+	end := len(runes)
+	for ; end > start && unicode.IsPunct(runes[end-1]); end-- {
+	}
+
+	return string(runes[start:end])
```

Comment:
I think something like this will work:

```suggestion
	return strings.TrimFunc(input, unicode.IsPunct)
```

Reply:
Nice, thanks! Interestingly, Claude doesn't seem to know about this.